tools/src/scie_pants/pants_version.py

```diff
@@ -99,6 +99,11 @@ def determine_find_links(
 def determine_tag_version(
     ptex: Ptex, pants_version: str, find_links_dir: Path, github_api_bearer_token: str | None = None
 ) -> ResolveInfo:
+    # `pants_version` should be in the format `major.minor.micro`, e.g., `2.17.0`.
+    # Raise an error if it does not follow this format.
+    if not pants_version.count(".") == 2:
```

Comment:
FYI, a dev release has a format with more `.`s, so we'll need to refine this somewhat to support them. For instance: https://github.com/pantsbuild/pants/releases/tag/release_2.20.0.dev3

Reply:
I just improved the error message and added a test. Please take a look! Thank you.